src/repl.jl

```diff
@@ -6,8 +6,8 @@ function RunDebugger(stack, repl = Base.active_repl, terminal = Base.active_repl
 
     # Setup debug panel
     panel = LineEdit.Prompt(promptname(state.level, "debug");
-        prompt_prefix="\e[38;5;166m",
-        prompt_suffix=Base.text_colors[:white],
+        prompt_prefix = "\e[38;5;166m",
+        prompt_suffix = Base.text_colors[:normal],
```

Comment:
I think this makes the tests fail.

Reply:
Probably. Added a few more repl tweaks and fixed the tests.